### ISSUE TITLE
Nuget: Unable to resolve dependency 'NETStandard.Library'. targeting .net 4.6.1

### ISSUE THREAD

**Issue by eddiegroves** [lunet-io/markdig#3]
```
Install-Package Markdig
Attempting to gather dependency information for package 'Markdig.0.1.0' with respect to project 'ConsoleApplication1', targeting '.NETFramework,Version=v4.6.1'
Attempting to resolve dependencies for package 'Markdig.0.1.0' with DependencyBehavior 'Lowest'
Install-Package : Unable to resolve dependency 'NETStandard.Library'.
```

Successful work around is to ignore the dependency:

```
Install-Package Markdig -IgnoreDependencies
```

**Comment by xoofx** [comment]
Ah, probably messed  up the project.json while upgrading to dotnet rc2, being away for few days, will be only able to fix it early next week, thanks for the workaround!

**Comment by xoofx** [comment]
Thanks, this should be fixed by commit fb559af (version 0.1.1)
